package.json

```diff
@@ -21,7 +21,8 @@
     "acceptance-test": "lerna exec --scope @laconia/acceptance-test npm run test",
     "posttest": "npm run test:types && npm run test:jslint && npm run test:packagelint",
     "publish:coverage": "cat ./coverage/lcov.info | coveralls",
-    "commit": "npx git-cz"
+    "commit": "npx git-cz",
+    "prepare": "husky install"
```

Comment:
@geoffdutton, @ceilfors  should it be added to postInstall?

Reply:
Correct, I read this article while implementing.